seatunnel-core/seatunnel-flink-starter/seatunnel-flink-13-starter/src/main/bin/start-seatunnel-flink-13-connector-v2.sh

```diff
@@ -43,6 +43,25 @@ if [ -f "${CONF_DIR}/seatunnel-env.sh" ]; then
     . "${CONF_DIR}/seatunnel-env.sh"
 fi
 
+if [ ! -f "${APP_DIR}/runtime.tar.gz" ];then
```

Comment:
If not only one job are submitted at same time, will the files between different tasks conflict?

Reply:
Yes, you are right. If user use application mode, runtime.tar.gz should be prepared by user first, this step we will show it in new version docs.